### PR TITLE
Information about the 3.0.0-RC1 release

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: src/site/xdoc/server/release-notes.xml [hunk 1]
```diff
@@ -35,6 +35,31 @@
...
+            <p>Released May 2017</p>
+
+            <p>The Apache James project is happy to announce the release
+                of version 3.0.0-RC1 of its modular mail server.</p>
+
+            <p>We thank all Users for their feedback and tests and we encourage anyone to
+                <a href="http://james.apache.org/download.cgi#Apache_James_Server">download</a>
+                and discover our RC1 release.</p>
+
+            <ul>What have been done in James 3.0.0-RC1 :
```
Comment (chibenwa) [inline src/site/xdoc/server/release-notes.xml]: Couldn't this be duplicated in the post?

Comment (aduprat) [inline src/site/xdoc/server/release-notes.xml]: It was not in the post for previous releases.

**Comment by aduprat** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**